src/routes/Employee.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import {
+  createEmployee,
+  readAllEmployee,
+  readEmployee,
+  updateEmployee,
+  deleteEmployee,
+} from "../controllers/Employee";
```

Comment:
change all the related address to absolute address
fix other first and i will give you instruction on this